composer.json

```diff
@@ -15,14 +15,13 @@
     ],
     "require": {
         "php": ">=5.4.0",
-        "illuminate/support": "~4.2|~5.0"
+        "illuminate/support": "~5.0"
```

Comment:
Better to make it `5.1.*`. No point in supporting 5.0.x as it's no longer supported by laravel anymore.